client/my-sites/stats/controller.jsx

```diff
@@ -176,6 +178,20 @@ export default {
 
 	site: function( context, next ) {
 		const { params: { site_id: givenSiteId }, query: queryOptions, store } = context;
+
+		if ( 'simplePaymentsEmailTour' === get( queryOptions, 'tour' ) ) {
+			if ( ! isDesktop() ) {
+				context.store.dispatch(
+					recordTracksEvent( 'calypso_simple_payment_email_tour', { source: 'mobile' } )
+				);
+				window.location.href = 'https://en.support.wordpress.com/simple-payments/';
```

Comment:
I think it would be better if we redirected to `https://support.wordpress.com/simple-payments/` instead of hardcoding the locale.

Reply:
Nice catch, thanks!